### ISSUE TITLE
Error when installing vim keybindings plugin on fresh install on macos

### ISSUE THREAD

**Issue by jakemanger** [inkdropapp/inkdrop-vim#43]
I encounter this error when installing the vim keybindings:
<img width="343" alt="image" src="https://github.com/inkdropapp/inkdrop-vim/assets/52495554/428998fe-bf36-4cbc-8dd5-de4acc8cfc91">

Here's my inkdrop version:
<img width="281" alt="image" src="https://github.com/inkdropapp/inkdrop-vim/assets/52495554/ba017fdd-3424-448b-9dbf-75cd4216544d">

Do you know what the problem is?

**Comment by craftzdog** [comment]
Have you installed other plugins?
What is the callstack of the error in Developer Tools Console?

**Comment by jakemanger** [comment]
No, this is a fresh install on a new m2 macbook pro. Here's the developer tools console output after starting inkdrop:
<img width="555" alt="image" src="https://github.com/inkdropapp/inkdrop-vim/assets/52495554/fd605649-eecc-4e12-8303-7c7a10d1f555">
<img width="550" alt="image" src="https://github.com/inkdropapp/inkdrop-vim/assets/52495554/c577f912-a17d-471f-93a2-a0b75c990f43">

**Comment by craftzdog** [comment]
Thanks for the information.
It looks like the plugin fails to load the CodeMirror module, which is strange.
v5.5.3 has been stable and I haven't ever got such an error report from other people so far.
You are on M2 MacBook.
Have you installed the binary for Apple Silicon (not Intel)?

**Comment by jakemanger** [comment]
Yes, I am using the Apple Silicon arm64 binary and also just tried version 5.5.2 (by changing the numbers in the download link) with no change in behaviour. I also tried installing from ipm, however it still causes the same error. I have actually used codemirror on my own projects before. Is there a chance that inkdrop is sharing the version of codemirror that I'm using or any other npm packages? Is there a older or newer version of inkdrop I could try?

PS: thanks for the help. I really love your videos and desperately want vim mode in inkdrop because I too am so used to the keybindings when moving around. Without it I end up with a bunch of ":wq" and "kkkjjjkjjk" in my documents 😅
